libs/blocks/icon-block/icon-block.js

```diff
@@ -39,8 +39,18 @@ const iconBlocks = {
   },
 };
 
+//checking if block is part of xx-up and is inline variant
+function upAndInline(el) {
+  const isInline = el.classList.contains('inline');
+  if(!isInline) return false;
+  const sectionMetadata = el.parentElement?.querySelector('.section-metadata');
+  if(!sectionMetadata) return false;
```

Comment:
nit, could make a bit more concise.  Maybe something like? 

```suggestion
  const isInlineAndSectionMetadata = el.classList.contains('inline') && el.parentElement?.querySelector('.section-metadata')
  if(!isInlineAndSectionMetadata) return false;
```

Reply:
Yeah, this looks better. Thank you.